_includes/nav.html

```diff
@@ -96,4 +96,13 @@
       {%- endunless -%}
     {%- endif -%}
   {%- endfor -%}
+  {%- assign nav_external_links = site.nav_external_links -%}
+  {%- for node in nav_external_links -%}
+    <li class="nav-list-item external">
+      <a href="{{ node.url | absolute_url }}" class="nav-list-link external">
+        {{ node.title }}
+        {% unless node.hide_icon %}<svg viewBox="0 0 24 24"><use xlink:href="#svg-external-link"></use></svg>{% endunless %}
```

Comment:
Since this icon doesn't add any semantic meaning, we should hide it from a screenreader. I think in this case, we want to add 

```suggestion
        {% unless node.hide_icon %}<svg aria-hidden="true" viewBox="0 0 24 24"><use xlink:href="#svg-external-link"></use></svg>{% endunless %}
```

For more, [this blog post by CSS Tricks](https://css-tricks.com/accessible-svg-icons/#aa-the-icon-is-decorative) is a good summary.

Reply:
Nice points about accessibility. Correct me if I'm wrong -- but doesn't this icon convey that this is an external link, thus has semantic meaning?

Reply:
You know what - you're completely right, that slipped my mind. Hm, in that case, I think having some screenreader-only/aria text that says "(external link)" would make the most sense. 

It seems like there's no consensus on exactly what it should say. If you have a suggestion instead of "(external link)", please go for it instead!

Precedent:

- [W3 WCAG20 spec](https://www.w3.org/TR/WCAG20-TECHS/G200.html) saying that advanced warning is necessary
- [deque checklist on links](https://dequeuniversity.com/checklists/web/links)
- [WebAIM article](https://webaim.org/techniques/hypertext/hypertext_links)

Reply:
Thanks for the response! The [`<title>` element](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/title) can also be used to provide a text description for svg (it doesn't work for `<symbol>`, where it is right now, so it'd need to be moved inside the `<svg>` container), which would also be shown as a tooltip on most browsers. Not sure if that's preferable over `aria-label`.

An issue I found is that there are two layers of `svg` elements (one inside the `<symbol>` definition, another outside the `<use>` reference), resulting in two `diagram` elements on the accessibility tree. Not sure what's the best way to describe them for screenreaders -- perhaps both `aria-label` for the `<svg>` outside of `<use>` and `<title>` for the `<svg>` inside of `<symbol>`?

![image](https://user-images.githubusercontent.com/15277496/177390568-ecdf1b7e-3748-4a51-9037-538c77ff8f78.png)

Reply:
Upon some experiments the `<svg>` element under the `<symbol>` definition isn't actually necessary. I will just remove that `<svg>` and go with providing a text description for the remaining `<svg>` via `aria-label`.